src/Microsoft.DocAsCode.Build.Incrementals/DependencyGraph.cs

```diff
@@ -59,6 +59,13 @@ public IEnumerable<string> Keys
             get { return _dictionary.Keys; }
         }
 
+        public SortedSet<string> GetDirectDependency(string key)
```

Comment:
how?

---
In reply to: [75277009](https://github.com/dotnet/docfx/pull/618#discussion_r75277009) [](ancestors = 75277009)